docs-v2/api-reference/integration/get.mdx

```diff
@@ -6,12 +6,10 @@ openapi: 'GET /config/{providerConfigKey}'
 <ResponseExample>
   ```json Example Response
 {
```

Comment:
@bastienbeurier no reason to wrap a single element in an array so I updated this.